### ISSUE TITLE
Extra indentation after line with multiple assignments

### ISSUE THREAD

**Issue by mihaipopescu** [uncrustify/uncrustify#1277]
Expected versus output diff:
---
```diff
diff --git "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\60025-UNI-19894.cpp" "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\60025-UNI-19894.cpp"
index 2d7e481..53d653f 100644
--- "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\60025-UNI-19894.cpp"
+++ "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\60025-UNI-19894.cpp"
@@ -1,16 +1,16 @@
 //It is applying double indentation
 m_ApplicationView = AppC::ApplicationView::GetForCurrentView();
 m_ViewConsolidateEvtToken = m_ApplicationView->Consolidated +=
-    ref new AppC::TypedEventHandler<AppC::ApplicationView^, AppC::ApplicationViewConsolidatedEventArgs^>(this, &FrameworkView::InternalOnViewConsolidated);
+        ref new AppC::TypedEventHandler<AppC::ApplicationView^, AppC::ApplicationViewConsolidatedEventArgs^>(this, &FrameworkView::InternalOnViewConsolidated);
 
 m_WindowActivatedEvtToken = m_CoreWindow->Activated +=
-    ref new AppC::TypedEventHandler<AppC::CoreWindow^, AppC::WindowActivatedEventArgs^>(this, &FrameworkView::InternalOnWindowActivated);
+        ref new AppC::TypedEventHandler<AppC::CoreWindow^, AppC::WindowActivatedEventArgs^>(this, &FrameworkView::InternalOnWindowActivated);
 
 m_SizeChangedEvtToken = m_CoreWindow->SizeChanged +=
-    ref new AppC::TypedEventHandler<AppC::CoreWindow^, AppC::WindowSizeChangedEventArgs^>(this, &FrameworkView::InternalOnWindowSizeChanged);
+        ref new AppC::TypedEventHandler<AppC::CoreWindow^, AppC::WindowSizeChangedEventArgs^>(this, &FrameworkView::InternalOnWindowSizeChanged);
 
 m_VisibilityChangedEvtToken = m_CoreWindow->VisibilityChanged +=
-    ref new AppC::TypedEventHandler<AppC::CoreWindow^, AppC::VisibilityChangedEventArgs^>(this, &FrameworkView::InternalOnWindowVisibilityChanged);
+        ref new AppC::TypedEventHandler<AppC::CoreWindow^, AppC::VisibilityChangedEventArgs^>(this, &FrameworkView::InternalOnWindowVisibilityChanged);
 
 m_WindowClosedEvtToken = m_CoreWindow->Closed +=
-    ref new AppC::TypedEventHandler<AppC::CoreWindow^, AppC::CoreWindowEventArgs^>(this, &FrameworkView::InternalOnWindowClosed);
+        ref new AppC::TypedEventHandler<AppC::CoreWindow^, AppC::CoreWindowEventArgs^>(this, &FrameworkView::InternalOnWindowClosed);
```

Debug support files:
---
[60025-UNI-19894.cpp](https://gist.githubusercontent.com/mihaipopescu/34f360b13353a173efbb1f3651c581fe/raw/2d7e481e407b5a5ef30cc0a2cb0aba30284776ea/60025-UNI-19894.cpp)
[60025-UNI-19894.cpp.log](https://gist.githubusercontent.com/mihaipopescu/34f360b13353a173efbb1f3651c581fe/raw/c5dd75844e187a291005529aa5c77410397383b4/60025-UNI-19894.cpp.log)
[60025-UNI-19894.cpp.unc](https://gist.githubusercontent.com/mihaipopescu/34f360b13353a173efbb1f3651c581fe/raw/ef1bcc7e20b26c80b296e34218c705620b99edd2/60025-UNI-19894.cpp.unc)
[Uncrustify.Cpp.cfg](https://gist.githubusercontent.com/mihaipopescu/34f360b13353a173efbb1f3651c581fe/raw/23d62d9dba5caf352fdc1793792e98eed4b81929/Uncrustify.Cpp.cfg)
[UNI-19894.cpp](https://gist.githubusercontent.com/mihaipopescu/34f360b13353a173efbb1f3651c581fe/raw/ef5fe7cf30aeaf32f77375abcb2a7bb3373f2570/UNI-19894.cpp)

Disclaimer:
---
Note: This is a generated issue from our internal issue tracker.
Expect a dry description which could be insufficient to properly understand the issue.
On the other hand, the support files should be sufficient to easily reproduce and debug the issue,
but please comment if it's still unclear and I will get back with clarifications.
See main discussion thread here #1223.

**Comment by CDanU** [comment]
min cfg
```sql
# Uncrustify-0.65-89-476df4d6
input_tab_size                  = 4
indent_columns                  = 4
indent_with_tabs                = 0
indent_align_assign             = false
# option(s) with 'not default' value: 4
#
```

**Comment by mihaipopescu** [comment]
I should really insert `option_reducer` script in the pipeline :)

**Comment by gmaurel** [comment]
Fixed at 2c412d8929c2c11e50660ad82e95271831e4a98e, Dec 17 18:33:16 2017
